constructs/Microsoft.Authorization/roleAssignments-multiRolesMultiPrincipals/readme.md

```diff
@@ -2,6 +2,14 @@
 
 This module deploys Role Assignments.
 
+## Navigation
```

Comment:
Do we need module readmes in the costructs folder?

Reply:
@eriqua I think we do. But regardless it's something we can discuss outside of this PR. (I just re-ran the generation to make sure all modules compile)